src/main/fc/fc_init.c

```diff
@@ -196,52 +197,58 @@ static IO_t busSwitchResetPin        = IO_NONE;
 
 void spiPreInit(void)
 {
-#ifdef USE_ACC_SPI_MPU6000
+#ifdef GYRO_1_CS_PIN
+    spiPreInitCs(IO_TAG(GYRO_1_CS_PIN));
+#endif
+#ifdef GYRO_2_CS_PIN
+    spiPreInitCs(IO_TAG(GYRO_2_CS_PIN));
+#endif
+#ifdef MPU6000_CS_PIN
     spiPreInitCs(IO_TAG(MPU6000_CS_PIN));
 #endif
-#ifdef USE_ACC_SPI_MPU6500
+#ifdef MPU6500_CS_PIN
```

Comment:
I think this is not right.

USE and PIN are different.

Reply:
yes, 100% agreed. however if a target defines the pin then it MUST be initialised, even if the sensor detection code is compiled out.

Reply:
I think PIN were converted to USE at some point in the past, to distinguish USE and PIN. I forgot what the reason was, though. May be a provisioning for drivers that require additional conditional inside the USE, or handling multiple pins.

Reply:
Conversation at slack:
> basically, if you don't initialise all CS pins then the device, if it's unused but present on the PCB will be ACTIVE and start responding to data on the SPI MOSI/MISO lines that is meant for another device.

I'm okay with the change now.